_summaries/dai2019transformer_1.md

```diff
@@ -0,0 +1,44 @@
+---
+layout: summary
+title: Summary
+paper: dai2019transformer
+# Please fill out info below
+author: slycane9
+score: 9
+---
+
+## Core Idea
+The authors developed a new Transformer-XL model that had two features that separated itself from other "Vanilla" Transformers of the time in regards to creating Language Models.
```

Comment:
I would go ahead and list what the separates Transformer-XL from the traditional Transformer is here. I would also mention that the development of this model is in response to RNNs and LSTMs having context fragmentations, and also explain what this is.

Also I think it would be beneficial to the reader is you expressed here that XL stands for extra long and not extra large, as this could cause confusion later.